omniscidb/Tests/GroupByTest.cpp

```diff
@@ -389,12 +376,18 @@ int main(int argc, char** argv) {
   logger::LogOptions log_options(argv[0]);
   log_options.max_files_ = 0;  // stderr only by default
   desc.add(log_options.get_options());
+  desc.add_options()("cpu-only",
```

Comment:
It would be better to use `ConfigBuilder` here and in all other places where it provides all the required options. That would enable the whole set of options, not only `cpu-only` one.